test/itkImageToTorchImageFilterTest.cxx

```diff
@@ -224,6 +231,59 @@ int itkImageToTorchImageFilterTest(int argc, char *argv[])
     typename FilterType::Pointer filter = FilterType::New();
   }
 
+  // Copy an image from an itkImage to an itkTorchImage and back.
+  //
+
+  // We will need a random number generator.
+  using UniformGeneratorType = itk::Statistics::MersenneTwisterRandomVariateGenerator;
+  UniformGeneratorType::Pointer uniformGenerator = UniformGeneratorType::New();
+  uniformGenerator->Initialize( 20200925 );
+
+  // The itkImage and itkTorchImage types we are testing.
+  using PixelType = uint8_t;
+  constexpr int ImageDimension = 2;
+  using InputImageType = itk::Image< PixelType, ImageDimension >;
+  using OutputTorchImageType = itk::TorchImage< PixelType, ImageDimension >;
+
+  // Build an itkImage
```

Comment:
This part of code might be replaced by [RandomImageSource](https://itk.org/ITKExamples/src/Core/TestKernel/GenerateRandomImage/Documentation.html).

Reply:
That looks useful!  The next time that I write a test ....